man/fi_psm.7.md

```diff
@@ -21,6 +21,18 @@ tag-matching message queue functions and the Active Message functions
 to support a variety of libfabric data transfer APIs, including tagged
 message queue, message queue, RMA, and atomic operations.
 
+The *psm2* provider runs over the PSM 2.0 interface that is
```

Comment:
Good idea, will do.

From: Jeff Squyres [mailto:notifications@github.com]
Sent: Wednesday, August 26, 2015 12:40 PM
To: ofiwg/libfabric <libfabric@noreply.github.com>
Cc: Xiong, Jianxin <jianxin.xiong@intel.com>
Subject: Re: [libfabric] prov/psm: add PSM2 support (#1242)


In man/fi_psm.7.md<https://github.com/ofiwg/libfabric/pull/1242#discussion_r38025391>:

> @@ -21,6 +21,18 @@ tag-matching message queue functions and the Active Message functions

>  to support a variety of libfabric data transfer APIs, including tagged

>  message queue, message queue, RMA, and atomic operations.

>

> +The *psm2* provider runs over the PSM 2.0 interface that is

You might want to add a man/man7/fi_psm2.7 file that just contains

.so man7/fi_psm.7

That way, people can do man fi_psm or man fi_psm2, and they'll see the same man page.

—
Reply to this email directly or view it on GitHub<https://github.com/ofiwg/libfabric/pull/1242/files#r38025391>.